distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -1204,6 +1204,98 @@ public void testNoEnforceValidityErrorsOrWarnings() throws Exception {
     }
 
     @Test
+    public void testQueryByErrorFailedValidators() throws Exception {
+        getServiceManager().startFeature(true,
+                "catalog-plugin-metacard-validation",
+                "sample-validator");
```

Comment:
Sorry, I may have been confusing; the "catalog-plugin-metacard-validation" is automatically started, you still need to start the "sample-validator"